TerminalReuse.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Perhaps we should move all these executable files and scripts into a subfolder so they don't clutter the repository root?

Reply:
Good idea 👍🏻

Reply:
ugh, the paths to these script are used in settings and such, it will be a bit disruptive to move them

Reply:
Hm, that's unfortunate. Let's consider it for some other time, then.